cmd/repo-updater/shared/main.go

```diff
@@ -270,10 +270,15 @@ func watchSyncer(ctx context.Context, syncer *repos.Syncer, sched scheduler, gps
 // syncCloned will periodically list the cloned repositories on gitserver and
 // update the scheduler with the list.
 func syncCloned(ctx context.Context, sched scheduler, gitserverClient *gitserver.Client, store repos.Store) {
+	sleepDuration := 10 * time.Second
```

Comment:
nit: Since it's not designed to change (didn't validate if it's possible doing so):
```suggestion
	const sleepDuration = 10 * time.Second
```